Makefile

```diff
@@ -34,9 +34,15 @@ $(LINUX):
 $(WINDOWS):
 	env CGO_ENABLED=0 GOOS=windows GOARCH=amd64 go build -o $(WINDOWS) -ldflags "-X main.date=$(DATE) -X main.vers=$(VERS) -X main.hash=$(HASH)" ./cmd/$(BIN)
 
-test:
+test: test-binary test-node
+
+test-binary:
 	go test -race -cover -coverprofile=coverage.out ./...
 
+test-node:
```

Comment:
Nice. We should add something similar for Quarkus and Go too.

Reply:
For sure. I just added https://github.com/boson-project/faas/issues/70

Reply:
Definitely nice to have the templates included in the testing pass